css/Z_INDEX.md

```diff
@@ -14,5 +14,8 @@ amp-image-lightbox                          |   1000      |   extensions/amp-ima
 amp-live-list > [update]                    |   1000      |   extensions/amp-live-list/0.1/amp-live-list.css
 amp-user-notification                       |   1000      |   extensions/amp-user-notification/0.1/amp-user-notification.css
 .-amp-image-lightbox-trans                  |   1001      |   extensions/amp-image-lightbox/0.1/amp-image-lightbox.css
+.-amp-lightbox-viewer-mask                  |  2147483644 |   extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css
```

Comment:
I think these are ordered by the z-index. So these should be at the end of the file.